Ginger/GingerCoreNET/LiteDBFolder/LiteDbReportBase.cs

```diff
@@ -205,9 +225,24 @@ public class LiteDbActivity : LiteDbReportBase
     {
         public string ActivityGroupName { get; set; }
         public List<LiteDbAction> ActionsColl { get; set; }
+        //public List<Tuple<string, string>> ChildExecutableItemsCount { get; set; }
+        //public List<Tuple<string, string>> ChildExecutedItemsCount { get; set; }
+        //public List<Tuple<string, string>> ChildPassedItemsCount { get; set; }
+
+        public Dictionary<string, int> ChildExecutableItemsCount { get; set; }
+        public Dictionary<string, int> ChildExecutedItemsCount { get; set; }
+        public Dictionary<string, int> ChildPassedItemsCount { get; set; }
+
         public LiteDbActivity()
         {
             ActionsColl = new List<LiteDbAction>();
+            //ChildExecutableItemsCount = new List<Tuple<string, string>>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4549121)